xml/schema/Member/MembershipType.xml

```diff
@@ -115,7 +115,8 @@
   <field>
     <name>minimum_fee</name>
     <title>membership Type Minimum Fee</title>
-    <type>decimal</type>
+    <type>varchar</type>
+    <length>512</length>
```

Comment:
When you edit membership min_amount it creates a default price set / updates one with a value for the membership type - so that goes there. Not sure why civicrm_price_field_value.amount  would be  varchar!